### ISSUE TITLE
Specify path to compass?

### ISSUE THREAD

**Issue by nfriedly** [docpad/docpad-plugin-sass#33]
Hi, I'm trying to deploy a site that uses this to Bluemix, which is basically IBM's version of Heroku.

It's using a node.js buildpack, so it automatically installs things from package.json, but ignores the Gemfile.

Next,  I tried changing the start command `gem install compass && node server.js`, but it choked because the gems directory is read-only. (But the good news is that ruby is installed).

After that, I copied the ruby gems into my app and configured the `sassPath` and `scssPath` options - that got me even further, but it choked when I tried to find compass in the system ruby gems directory.

So, is there a way to specify the path to compass in addition to sass and scss? Or, any other way you can think of to solve this?

**Comment by balupton** [comment]
good by me, PR welcome

**Comment by balupton** [comment]
actually, I just ran into this issue, turns out is is actually caused to version incompatibilities between compass and sass. The new version of this plugin will output the error message if it is there.

**Comment by balupton** [comment]
fixed in v2.9.0

**Comment by balupton** [comment]
For instance, the fix for me was:

```
gem install sass --version 3.4.25
gem install compass --version 1.0.3
```
